smithy-model/src/test/resources/software/amazon/smithy/model/loader/valid/mixins/loads-mixins.smithy

```diff
@@ -3,20 +3,32 @@ $version: "2.0"
 namespace smithy.example
 
 @mixin
-structure A {}
+structure A {
 
-structure B with [A] {}
+}
+
+structure B with [A] {
+
+}
 
 @mixin
-structure C {}
+structure C {
```

Comment:
Similar comment as inline input/ouptut I guess -- Is it possible to support single line structures?